WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -142,9 +142,9 @@ await foreach (var offendingAlices in CheckTxoSpendStatusAsync(round, cancel).Co
 					{
 						// This should never happen.
 
-						Logger.LogError($"{nameof(CoinVerifier)} has failed to verify all Alices({round.Alices.Count}).", exc);
+						Logger.LogError($"{nameof(CoinVerifier)} has failed to verify all Alices({round.Alices.Count}) for Round({round.Id}).", exc);
 						CoinVerifier.VerifierAuditArchiver.LogException(round.Id, exc);
-						round.EndRound(EndRoundState.AbortedWithError);
```

Comment:
We set `round.EndRound(EndRoundState.AbortedWithError);` in the outer catch block.